Source/Core/Core/HW/Memmap.cpp

```diff
@@ -347,4 +347,17 @@ bool IsRAMAddress(const u32 address, bool allow_locked_cache, bool allow_fake_vm
 	}
 }
 
+bool IsPhysical(const u32 address)
```

Comment:
This looks suspicious; nothing outside of the CPU memory access functions should care about IsPhysical in this sense.  Everything else masks off the top bits.